.github/workflows/test-app.yml

```diff
@@ -77,6 +77,12 @@ jobs:
           filters: |
             react:
               - 'packages/react/**'
+      
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        if: steps.changes.outputs.react == 'true'
+        with:
+          node-version: '16.x'
```

Comment:
This change has already been made. This was failing across all PRs and a fix was delivered last week.

Reply:
it looks identical I would not think its a problem